src-backbone/app/js/views/builder/builderHeaderView.js

```diff
@@ -19,9 +19,9 @@ module.exports = Marionette.ItemView.extend({
         'click @ui.downloadButton': '_download'
     },
 
-    _save: function() {
-        Helpers.delaySave(this, this._saveToServer);
-    },
+    _save: _.debounce(function(event) {
```

Comment:
this could be simplified to just `_.debounce(this.saveToServer, Config.INPUT_DELAY_BEFORE_SAVE)`